spec/rubocop/cop/metrics/method_length_spec.rb

```diff
@@ -189,4 +189,35 @@ def m
       RUBY
     end
   end
+
+  context 'when ExcludedMethods is enabled' do
```

Comment:
It's not exactly a boolean option, so the wording here can be improved a bit.

Reply:
FIxed.